x-pack/platform/plugins/shared/integration_assistant/public/components/create_integration/create_integration_assistant/steps/data_stream_step/data_stream_step.tsx

```diff
@@ -277,16 +278,34 @@ export const DataStreamStep = React.memo<DataStreamStepProps>(
                       </EuiText>
                       <EuiFlexGroup>
                         {celInputResult ? (
-                          <EuiFlexGroup>
-                            <EuiButton iconType="check" disabled data-test-subj="openApiConfigured">
-                              <EuiText size="xs">{i18n.OPEN_API_SPEC_BUTTON_CONFIGURED}</EuiText>
-                            </EuiButton>
-                            <EuiButtonEmpty
-                              iconType="pencil"
-                              onClick={() => setShowCelCreateFlyout(true)}
-                            >
-                              <EuiText size="xs">{i18n.EDIT_OPEN_API_SPEC_BUTTON}</EuiText>
-                            </EuiButtonEmpty>
+                          <EuiFlexGroup direction="column" gutterSize="s">
+                            <EuiFlexGroup>
+                              <EuiFlexItem grow={false}>
+                                <EuiButton
+                                  iconType="check"
+                                  color="success"
+                                  data-test-subj="openApiConfigured"
+                                >
+                                  <EuiText size="xs">
+                                    {i18n.OPEN_API_SPEC_BUTTON_CONFIGURED}
+                                  </EuiText>
+                                </EuiButton>
+                              </EuiFlexItem>
+                            </EuiFlexGroup>
+                            <EuiTitle size="xxxs">
+                              <h5>{integrationSettings?.apiSpecFileName}</h5>
+                            </EuiTitle>{' '}
```

Comment:
`{' '}` is this a leftover

Reply:
Weird, I don't remember doing that intentionally. Removed.